src/components/PageRouter/PageRouter.component.jsx

```diff
@@ -19,6 +20,7 @@ const PageRouter = () => {
         <Route exact path="/products" component={ProductList} />
         <Route exact path="/product/:productId" component={ProductDetail} />
         <Route exact path='/search' component={SearchResults} />
+        <Route exact path='/cart' component={ShoppingCart} />
         <Route path="*">
           <span>404 Not Found.</span>
         </Route>
```

Comment:
imo it is better to leave the definition of the routes alone. And in the parent component where you are using them handle what you have below to switch from home to products.
Something like this
![image](https://user-images.githubusercontent.com/12141670/144888438-8a49abf1-2df2-412c-b965-c5a3e8609a8e.png)
A router component as i mentioned is typically used just for the definition of the routes
```suggestion
const PageRouter = () => {
    
  return (
    <>
      <Switch>
        <Route exact path="/home" component={Home} />
        <Route exact path="/products" component={ProductList} />
        <Route exact path="/product/:productId" component={ProductDetail} />
        <Route exact path='/search' component={SearchResults} />
        <Route exact path='/cart' component={ShoppingCart} />
        <Route path="*">
          <span>404 Not Found.</span>
        </Route>
      </Switch>
    </>
  );
}

export default PageRouter;
```

Reply:
You can even use a map and then on a separate file define the routes. IE:
create a routes config file `routes-config.js`: 
```
export const routes = [
  { path: '/', component: Home, exact: true },
  { path: '/login', component: Login, exact: true },
  { path: '*', component: Home, exact: true },
];
```
and on the router itself
```
export const PageRouter = () => {
    
  return (
    <>
      <Switch>
       {
          routes.map({ route, component }) =>  {
            <Route exact path={route} component={component} />
         })
       }
        <Route path="*">
          <span>404 Not Found.</span>
        </Route>
      </Switch>
    </>
  );
};
```